powerauth-restful-security-spring-annotation/src/main/java/io/getlime/security/powerauth/rest/api/spring/provider/PowerAuthEncryptionProvider.java

```diff
@@ -79,4 +83,29 @@ public PowerAuthEncryptionProvider(PowerAuthClient powerAuthClient, HttpCustomiz
         }
     }
 
+    @Nonnull
+    @Override
+    public PowerAuthEciesEncryptorParameters getEciesEncryptorParameters(@Nullable String activationId, @Nonnull String applicationKey, @Nonnull String ephemeralPublicKey, @Nonnull String version, String nonce, Long timestamp) throws PowerAuthEncryptionException {
+        try {
+            final GetEciesDecryptorRequest eciesEncryptorRequest = new GetEciesDecryptorRequest();
+            eciesEncryptorRequest.setActivationId(activationId);
+            eciesEncryptorRequest.setApplicationKey(applicationKey);
+            eciesEncryptorRequest.setEphemeralPublicKey(ephemeralPublicKey);
+            eciesEncryptorRequest.setProtocolVersion(version);
+            eciesEncryptorRequest.setNonce(nonce);
+            eciesEncryptorRequest.setTimestamp(timestamp);
+            final GetEciesDecryptorResponse eciesEncryptorResponse = powerAuthClient.getEciesDecryptor(
```

Comment:
It looks like we implemented a whole facility for getting this encryption parameters from PAS, but actually we never use it. I mean, we have new API introduced in server's RESTful API, it's implementation in the server, new interfaces in a client library wrapping it and then, at this point, we finally call a different function.

Reply:
Indeed, removed.